### ISSUE TITLE
Accessibility: Cannot access video links with TalkBack

### ISSUE THREAD

**Issue by shortwavesurfer2009** [thunder-app/thunder#1472]
### Bug Description

A recent update allowed videos to be played directly from within Thunder, but now TalkBack cannot access video links. It just completely skips over them.

### Expected Behaviour

Links should have their own element so that they are clickable.

### Steps to Reproduce

1. Turn on Talkback from Settings Accessibility.
2. Open this link in thunder https://lemmy.ml/post/17313056
3. swipe right with one finger to move element by element
4. You will notice it goes directly from the collapse post button to the author of the post and completely skips the link.

### App Version

0.5.0-1

### Device

OnePlus Nord N200

### OS

Android 13

**Comment by hjiangsu** [comment]
Thanks for the report!

For these types of elements (links), is there a specific way you would like it to be read out? I believe right now, it'll read out the full URL. I'm not sure if that's what you are expecting but do let me know!

**Comment by shortwavesurfer2009** [comment]
> Thanks for the report!
> 
> For these types of elements (links), is there a specific way you would like it to be read out? I believe right now, it'll read out the full URL. I'm not sure if that's what you are expecting but do let me know!

Reading the entire link is expected.
